src/lib/registration/clients/scc_auto.rb

```diff
@@ -95,7 +95,7 @@ def import(settings)
         # merge reg code if not defined in the profile but
         # available from other sources
         product = Yast::AutoinstFunctions.selected_product
-        if product && !settings["reg_code"]
+        if product&.respond_to?(:short_name) && !settings["reg_code"]
```

Comment:
Nitpicking: this change might deserve a comment. It's not really understandable just from reading the code. Imagine someone trying to debug this in 2 years.